lesson07/assignment/vue-cli/src/App.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+<div id="app">
+ <div id="nav">
+    <b-navbar toggleable="lg" type="light" variant="light">
+       <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+       <b-collapse id="nav-collapse" is-nav>
+          <b-navbar-nav>
+             <b-nav-item  to="/" >Home</b-nav-item>
+             <b-nav-item  to="/about">About Us</b-nav-item>
+             <b-nav-item  to="/trips">Trips</b-nav-item>
+             <b-nav-item  to="/travel-photos">Travel Photos</b-nav-item>
+          </b-navbar-nav>
+       </b-collapse>
+    </b-navbar>
+ </div>
+ <div id="bod">
+ <router-view/>
+ </div>
+</div>
+
+<!--https://www.vuemastery.com/courses/next-level-vue/mixins/-->
```

Comment:
Keeping links to your sources or good resources is a good idea, when working on small or personal projects. I usually make a section in the README called "Resources" and keep them all there, so they're easy to find.